cmd/api/src/queries/graph.go

```diff
@@ -21,6 +21,7 @@ package queries
 import (
 	"bytes"
 	"context"
+	e "errors"
```

Comment:
This is a sign that we should refactor away from using our (admittedly) ancient internal `errors` package. Please update this file to remove the use of `github.com/specterops/bloodhound/errors`.

Reply:
updated file to use `errors`